scripts/test

```diff
@@ -4,4 +4,9 @@ set -e
 cd $(dirname $0)/..
 
 echo Running tests
-go test -cover -tags=test ./...
+go test --coverpkg=./pkg/... -coverprofile=coverage.out --count=1 ./pkg/...
```

Comment:
Near as I can tell, if you run `go test -cover` without specifying the other args it will ouptut the results in a pretty readable format (see [here](https://go.dev/blog/cover)). Can you elaborate on why you chose to output this to a file and then use awk to parse the file?

Reply:
`go test -cover` only show the coverage of each individual package which isn't that useful. The changes here show the total coverage across the entire repo.

I am unsure if you are also asking why we are using a file rather than piping to awk, but if you are, that is because a coverprofile is not the same as stdout and holds more statistical information about the test that ran.
https://go.dev/blog/cover#viewing-the-results

Reply:
Resolving - complete coverage could have it's uses eventually and since we don't want to introduce go/perl into the pipeline we are going to stick with awk.